### ISSUE TITLE
Refine schedule h4 endpoint

### ISSUE THREAD

**Issue by jason-upchurch** [fecgov/openFEC#4130]
### Summary
Need to refine the schedule H4 endpoint based on feedback from the Data-design studio issue (https://github.com/fecgov/fec-cms/issues/3432)

We need to refine prototype `disclosure.fec_fitem_sched_h4` created by #4112 and PR #4123. Iteration may be informed by related work on the design studio issue @JonellaCulmer is working. 

### Completion criteria
- [ ] endpoint implemented based on Data-design studio

### Technical considerations
- [ ] there may be database work done in parallel that could block testing of this issue until completed, e.g., need to add any relevant `tsvector` columns to the schedule h family of data tables

**Comment by cnlucas** [comment]
Data mapping https://docs.google.com/document/d/1_0SqudoNoAXBdauhDkhalW3LIYbyqPo_HU6aY8VOg4Y/edit

**Comment by cnlucas** [comment]
Closing in favor of https://github.com/fecgov/openFEC/issues/5361
